Source/Charts/Utils/ChartUtils.swift

```diff
@@ -175,7 +175,7 @@ extension CGContext
         }
     }
     
-    open func drawText(_ text: String, at point: CGPoint, anchor: CGPoint = CGPoint(x: 0.5, y: 0.5), angleRadians: CGFloat, attributes: [NSAttributedString.Key : Any]?)
+    public func drawText(_ text: String, at point: CGPoint, anchor: CGPoint = CGPoint(x: 0.5, y: 0.5), angleRadians: CGFloat, attributes: [NSAttributedString.Key : Any]?)
```

Comment:
Went with the Xcode suggested fix here.

Reply:
I can't remember now, sorry.

Reply:
> what's the error message for open?

quoted from [this](https://github.com/danielgindi/Charts/issues/4860#issue-1296502259) issue:
> Build error around CandleStickChartRenderer's and RadarChartRenderer's drawData(context: CGContext) where the IndexingIterator are not equivalent. Raw message:
>>error build: Referencing instance method 'makeIterator()' on 'Collection' requires the types 'IndexingIterator' and 'IndexingIterator' be equivalent

Reply:
The quoted issue doesn't seem to be relevant here. For me, the issue 

```
CandleStickChartRenderer.swift:41:59: error build: Referencing instance method 'makeIterator()' on 'Collection' requires the types 'IndexingIterator<ChartData>' and 'IndexingIterator<CandleChartData>' be equivalent
```

still was still there with this PR's branch. It was gone only after I made the required conversions: `radarData as ChartData` in RadarChartRenderer:55 and `candleData as ChartData` at CandleStickChartRenderer:41

I am using Xcode 14 beta 3

Reply:
+1

Reply:
+1

Reply:
The open vs public is not an error, but a build warning:

```
Non-'@objc' instance method in extensions cannot be overridden; use 'public' instead
```

Reply:
> The quoted issue doesn't seem to be relevant here. For me, the issue
> 
> ```
> CandleStickChartRenderer.swift:41:59: error build: Referencing instance method 'makeIterator()' on 'Collection' requires the types 'IndexingIterator<ChartData>' and 'IndexingIterator<CandleChartData>' be equivalent
> ```
> 
> still was still there with this PR's branch. It was gone only after I made the required conversions: `radarData as ChartData` in RadarChartRenderer:55 and `candleData as ChartData` at CandleStickChartRenderer:41
> 
> I am using Xcode 14 beta 3

This is the way!